elements/pfe-accordion/BaseAccordionHeader.scss

```
@@ -0,0 +1,113 @@
+  --pf-c-accordion__toggle--PaddingLeft: var(--pf-global--spacer--md, 1rem);
+  --pf-c-accordion__toggle--FontSize: var(--pf-global--FontSize--lg, 1rem);
+  --pf-c-accordion__toggle--FontFamily: var(--pf-global--FontFamily--redhat-updated--heading--sans-serif, "RedHatDisplayUpdated", helvetica, arial, sans-serif);
+  --pf-c-accordion__toggle--FontWeight: var(--pf-global--FontWeight--normal, 400);
+  --pf-c-accordion__toggle--active--BackgroundColor: var(--pf-global--BackgroundColor--200, #f0f0f0);
+  --pf-c-accordion__toggle--active-text--Color: var(--pf-global--link--Color, #0066cc);
+  --pf-c-accordion__toggle--active-text--FontWeight: var(--pf-global--FontWeight--semi-bold, 700);
+  --pf-c-accordion__toggle--expanded--before--BackgroundColor: var(--pf-global--primary-color--100, #0066cc);
+  --pf-c-accordion__toggle--expanded-icon--Rotate: 90deg;
+  --pf-c-accordion__toggle-text--MaxWidth: calc(100% - var(--pf-global--spacer--lg, 1.5rem));
+  --pf-c-accordion__toggle--before--Width: var(--pf-global--BorderWidth--lg, 3px);
+  --pf-c-accordion__toggle-icon--Transition: 0.2s ease-in 0s;
+  --pfe-icon--size: var(--pf-c-accordion__toggle--IconSize, 10px);
+
+  color: var(--pf-c-accordion__toggle--Color);
+  background-color: var(--pf-global--BackgroundColor--100);
+}
+
+#heading {
+  font-size: 100%;
+  padding: 0;
+  margin: 0;
+  font-weight: var(--pf-c-accordion__toggle--FontWeight);
+}
+
+button,
+a {
+  cursor: pointer;
+}
+
+.toggle,
+.toggle:before,
+.toggle:after {
+  padding: 0;
+  margin: 0;
+  background-color: var(--pf-c-accordion__toggle--BackgroundColor);
+}
+
+.icon {
+  transition: var(--pf-c-accordion__toggle-icon--Transition);
+}
+
+.toggle {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  border: 0;
+  padding:
+    var(--pf-c-accordion__toggle--PaddingTop)
+    var(--pf-c-accordion__toggle--PaddingRight)
+    var(--pf-c-accordion__toggle--PaddingBottom)
+    var(--pf-c-accordion__toggle--PaddingLeft);
+  font-family: var(--pf-c-accordion__toggle--FontFamily, inherit);
+  font-size: var(--pf-c-accordion__toggle--FontSize, inherit);
+  font-weight: var(--pf-c-accordion__toggle--FontWeight);
+  color: var(--pf-c-accordion__toggle--Color);
+}
+
+.toggle[aria-expanded="true"] {
+  --pf-c-accordion__toggle--after--BackgroundColor: var(--pf-c-accordion__toggle--expanded--before--BackgroundColor);
+}
+
+.toggle:after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  top: var(--pf-c-accordion__toggle--before--Top, 0);
+  width: var(--pf-c-accordion__toggle--before--Width);
+  background-color: var(--pf-c-accordion__toggle--after--BackgroundColor);
+}
+
+span {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  max-width: var(--pf-c-accordion__toggle-text--MaxWidth);
+}
+
+.toggle[aria-expanded="true"] .icon {
+  transform: rotate(var(--pf-c-accordion__toggle--expanded-icon--Rotate, 90deg));
```

I think we should already prefer transform properties:

```suggestion
  rotate: var(--pf-c-accordion__toggle--expanded-icon--Rotate, 90deg);
```